hardware/raspberrypi/bootmodes/README.md

```diff
@@ -2,23 +2,24 @@
 
 ## Introduction
 
-The Raspberry Pi has a number of different stages of booting. This document is meant to help explain how the boot modes work, and which ones are supported for Linux booting.
+The Raspberry Pi has a number of different stages of booting. This document explains how the boot modes work, and which ones are supported for Linux booting.
 
-* [Bootflow](bootflow.md): Boot sequence description
-* [SD card](sdcard.md): SD card boot description
-* [USB](usb.md): USB boot description
-  * [Device boot](device.md): Booting as a mass storage device
-  * [Host boot](host.md): Booting as a USB host
-    * [Mass storage boot](msd.md): Boot from Mass Storage Device (MSD)
-    * [Network boot](net.md): Boot from ethernet
+[Boot Sequence](bootflow.md)
 
-## Special bootcode.bin-only boot mode
+[SD card boot](sdcard.md)
 
-For the original Raspberry Pi and the Raspberry Pi 2 (based on the BCM2835 and BCM2836 devices), and in situations where the Pi 3 fails to boot, there is a new method of booting from one of the new boot modes (MSD or ethernet).
+[USB boot](usb.md) comprises the following two modes:
+* [Device boot](device.md): booting as a mass storage device
+* [Host boot](host.md): booting as a USB host using one of the following:
+  * [Mass storage boot](msd.md): boot from mass storage device
+  * [Network boot](net.md): Boot via Ethernet
```

Comment:
Should this also be a lower-case `b` for consistency?

Reply:
done 😊